starter-code/scripts/models/article.js

```diff
@@ -74,23 +77,24 @@
     webDB.execute(
       [
         {
-          // TODO: Insert an article instance into the database:
+          // DONE?: Insert an article instance into the database:
           // NOTE: this method will be called elsewhere after we retrieve our JSON
-          'sql': '', // <----- complete our SQL command here, inside the quotes.
+          'sql': 'INSERT INTO articles (title, category, author, authorUrl, publishedOn, body) VALUES (?, ?, ?, ?, ?, ?);',
+          // <----- complete our SQL command here, inside the quotes.
           'data': [this.title, this.author, this.authorUrl, this.category, this.publishedOn, this.body]
         }
       ]
-    );
+    ); console.log('executed');
   };
 
   Article.prototype.deleteRecord = function() {
     webDB.execute(
       [
         {
-          // TODO: Delete an article instance from the database based on its id:
+          // DONE?: Delete an article instance from the database based on its id:
           /* Note: this is an advanced admin option, so you will need to test
               out an individual query in the console */
-          'sql': '', // <--- complete the command here, inside the quotes;
+          'sql': 'DELETE FROM articles WHERE id', // <--- complete the command here, inside the quotes;
           'data': [this.id]
```

Comment:
Your SQL here is missing something. You need to specify the ID you want to delete with. Saying `WHERE id` is meaningless.